vagrant/provision.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export GO_VERSION=1.19
+export GO_VERSION="1.19.9"
```

Comment:
Does it make sense?

Reply:
1.19 ensures we are always the latest, and with `1.19.9` we may forget to update it

Reply:
Without it, v1.19 is downloaded, which is in fact v1.19.0.